css/main.css

```diff
@@ -1,3 +1,213 @@
 /* Add your CSS here */
 
 /* Dont' forget to link this file to your HTML in the <head> */
+
+header {
+  display: grid;
+  grid-template-columns: 20px 1fr 60px;
+  grid-template-rows: 50px;
+  align-items: center;
+  column-gap: 10px;
+  margin: 20px 15px 0 15px;
+  font-family: "Poppins", sans-serif;
+  color: black;
+  line-height: 2%;
+}
+
+main {
+  font-family: "Quicksand", sans-serif;
+  display: flex;
+  flex-direction: column;
+}
+.blue-box {
+  width: 20px;
+  height: 20px;
+  background-color: #0150fd;
+  grid-column: 1;
+  grid-row: 1;
+}
+
+.toggle-menu {
+  float: right;
+}
+
+@media screen and (min-width: 900px) {
+  .toggle-menu {
+    display: none;
+  }
+}
+
+.line-one {
+  border-top: 3px solid blue;
+  border-radius: 20px;
+  width: 10px;
+  margin-right: 30px;
+}
+.line-two {
+  border-top: 3px solid blue;
+  border-radius: 20px;
+  width: 20px;
+}
+.line-three {
+  border-top: 3px solid blue;
+  border-radius: 20px;
+  width: 10px;
+  margin-left: 30px;
+}
+
+h1 {
+  grid-column: 2;
+  grid-row: 1;
+  font-size: 1.5em;
+  font-family: Poppins;
+}
+
+h2 {
+  grid-column: 1 / span 2;
+  grid-row: 2;
+  font-size: 1.2em;
+  font-style: normal;
+  font-weight: 100;
+}
+
+.card {
+  width: 20em;
+  height: 28em;
+  text-align: center;
+  margin: auto;
+  margin-top: 20px;
+  background-color: #f4ece6;
+}
+
+.profile-img {
+  border-radius: 50%;
+  width: 150px;
+  margin-top: 20px;
+}
+.name {
+  font-size: 1.7em;
```

Comment:
Do you know how this font size is calculated? `px` or preferably `rem` are easier to work with and more predictable. `em` relies on the parent's font size so if this element moves, it will change.